addons/html_builder/static/src/builder/options/global_border_option.js

```diff
@@ -0,0 +1,30 @@
+import { Component } from "@odoo/owl";
+import { defaultBuilderComponents } from "../builder_components/default_builder_components";
+import { useDomState } from "../builder_components/utils";
+
+export class GlobalBorderOption extends Component {
```

Comment:
Global is unclear. Maybe BorderSelector ? BorderConfigurator ? Or something else :)